.github/workflows/main.yml

```diff
@@ -30,6 +30,8 @@ jobs:
         run: npm run check:metadata
       - name: Spellcheck
         run: npm run check:spelling
+      - name: Check orphaned pages
+        run: npm run check:orphan-pages
```

Comment:
Nit: please put this after the internal link checker. We want to order these by the likelihood of failure so that you get faster CI feedback.